src/pkg/sif/sifcore.go

```diff
@@ -5,9 +5,9 @@
   consult LICENSE file distributed with the sources of this project regarding
   your rights to use or distribute this software.
 */
-package image
+package sif
 
-// #cgo LDFLAGS: -lsycore -luuid
+// #cgo LDFLAGS: -lruntime -luuid
```

Comment:
Just wondering if there is any chance of bumping into some other lib called 'runtime' in a library path?